middlewares/levelCheckForQuestions.js

```diff
@@ -0,0 +1,20 @@
+import * as QuizModel from '../models/QuizModel'
+import * as UserModel from '../models/UserModel'
+import AppError from '../errors/AppError';
+
+export const levelCheck = async (req, res, next) => {
+    try {
+        const quiz = await QuizModel.getQuizByQuestionId(req.body.questionId);
+        const userLevel = req.user.level;
+        const possibleLevels = UserModel.userSchema.obj.level.enum
+        const accessLevel = 'quizer'
```

Comment:
Please create and use env variable to control access. Maybe in future this might change.  Also access level should be moderator.